models/char_cnn/__main__.py

```diff
@@ -115,6 +117,14 @@ def evaluate_dataset(split_name, dataset_cls, model, embedding, loader, batch_si
         if args.cuda:
             model.cuda()
 
+    if not args.trained_model:
+        save_path = os.path.join(args.save_path, dataset_map[args.dataset].NAME)
+        try:
+            os.makedirs(save_path)
```

Comment:
Is `os.makedirs(save_path, exist_ok=True)` sufficient?

Reply:
Fixed this in dc3ac773bdb101af8494a1321e30c0ca720d0ae3